packages/react/src/patterns/blocks/ContentGroupPictograms/__stories__/ContentGroupPictograms.stories.js

```diff
@@ -109,11 +119,7 @@ storiesOf('Patterns (Blocks)|ContentGroupPictograms', module)
           'Item 3 Copy (required)',
           'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean et ultricies est. Mauris iaculis eget dolor nec hendrerit. Phasellus at elit sollicitudin, sodales nulla quis, consequat libero.'
         ),
-        cta: object('Item 3 Link', {
-          type: 'local',
-          href: 'https://www.example.com',
-          copy: 'Lorem ipsum dolor',
-        }),
+        cta: toggleCta(boolean('Item 3 Cta')),
```

Comment:
don't you have to specify a boolean value?

`cta: toggleCta(boolean('Item 3 Cta', false))`

Reply:
I've added it on the latest commit, Thanks!